menstoreproject/menstoreproject/urls.py

```diff
@@ -18,11 +18,11 @@
 from django.conf import settings
 from django.conf.urls.static import static
 
-from catalog.views import index
+from catalog.views import ProductList
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path('', index, name='home'),
+    path('', ProductList.as_view(), name='home'),
```

Comment:
no, no, no. its a root url file. here you just add APP rout.  
path('', include('catalog.urls')), is enough.